lwgenerate/aws/aws.go

```diff
@@ -1050,6 +1050,9 @@ func createCloudtrail(args *GenerateAwsTfConfigurationArgs) (*hclwrite.Block, er
 			attributes["consolidated_trail"] = true
 		}
 		// S3 Bucket attributes
+		if args.CloudtrailName != "" {
```

Comment:
Why adding this? Will the following `if args.CloudtrailUseExistingTrail` work?

Reply:
Because we want the user to define the *new* cloudtrail name. Existing go-sdk is using "lacework-cloudtrail" as default new cloudtrail name and we cannot change it. 

There is a bug happen when user failed to delete the old "lacework-cloudtrail" and the new cloudtrail has to be a different name

Reply:
Isn't the code below (line 1054) `attributes["cloudtrail_name"] = args.CloudtrailName` using the passed in `args.CloudtrailName`?

Reply:
I get your point. I'll remove the line. `attributes["cloudtrail_name"]` should not depends on`args.CloudtrailUseExistingTrail `